### PR TITLE
Add ldas-tools-diskcacheapi-swig

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for ldas-tools-diskcacheAPI-swig, the bindings package for [ldas-tools-diskcacheapi](https://github.com/conda-forge/ldas-tools-diskcacheapi-feedstock/) (already forged).

@emaros, can you post a comment to indicate you are ok being named as co-maintainer.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x]  Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ldas-tools-diskcacheapi-swig/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```
Comment (chrisburr) [inline recipes/ldas-tools-diskcacheapi-swig/build.sh]: I think you need this to make sure it fails when there is an error
```suggestion
set -e

```

File: recipes/ldas-tools-diskcacheapi-swig/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    - {{ compiler('cxx') }}
+    - cmake >=3.6
+    - pkg-config  # [not win]
+    - make  # [not win]
+    - swig 3
+  host:
+    - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+    - ldas-tools-diskcacheapi >={{ ldas_tools_diskcacheapi_version }}
+  run:
+    - ldas-tools-diskcacheapi >={{ ldas_tools_diskcacheapi_version }}
```
Comment (scopatz) [inline recipes/ldas-tools-diskcacheapi-swig/meta.yaml]: need outputs here

Comment (duncanmmacleod) [reply recipes/ldas-tools-diskcacheapi-swig/meta.yaml]: @scopatz, are you just asking me to move the top-level `test` section to below the `outputs` section, or do you mean something else?

**Comment by duncanmmacleod** [comment]
ping @emaros, please post to indicate that you are happy to be named as co-maintainer, thanks.

**Comment by emaros** [comment]
> ping @emaros, please post to indicate that you are happy to be named as co-maintainer, thanks.

I am willing to be named as co-maintainer.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**